src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -1111,6 +1111,44 @@ def setArchDims(self, p1, p2):
             if self.dim2.Distance:
                 self.dim2.on()
 
+    def get_cursor_size(self):
+        # TODO Unfortunately, there's no way to get the cursor size in Qt
+        # Either provide platform-specific implementation or make it a user preference
+        # This should be in device-independent pixels
+        return 32
```

Comment:
If you think it's useful, we can define a parameter to help people tweak it in their system.
```py
param = App.ParamGet("User parameter:BaseApp/Preferences/Mod/Draft")
return param.GetInt("Draft_cursor_size", 32)
```

Reply:
I'd like it to be more general. For example, somewhere near the icon size. Or maybe we can just use the icon size. So that the cursor size would be relative to the icon size.

Ideally, I'd like it to be platform specific, or even create an issue in the Qt bug tracker.